contentcuration/contentcuration/static/js/edit_channel/uploader/views.js

```diff
@@ -303,7 +256,7 @@ var EditMetadataView = BaseViews.BaseEditableListView.extend({
     this.$("#metadata_prerequisites_btn").css("display", (is_individual && (has_files || is_exercise)) ? "inline-block" : "none");
     this.$("#metadata_questions_btn").css("display", (is_exercise) ? "inline-block" : "none");
     if(!is_individual){
-      this.render_details();
```

Comment:
Out of curiosity, what motivated you to switch to bootstrap tabs for this?

Reply:
Since it's already built in and we use it for the admin page, it didn't really make sense to have a whole separate implementation taking up a lot of lines in an already 1000+ line file